vdc/k8s-config.yaml

```diff
@@ -271,3 +271,55 @@ spec:
   resources:
     requests:
       storage: 10Mi
+---
+apiVersion: v1
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: test
+  name: delete-test-pvc
+rules:
+- apiGroups: ["*"]
```

Comment:
Is the "core" apiGroup sufficient? https://v1-10.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#persistentvolumeclaim-v1-core

If so, all could be `apiGroups: [""]` although I'm not really sure if that's clearer than apiGroups: ["core"]

Reply:
setting it to `""` works, so I made that change, seems pretty standard to use `""` in the docs.